pkgs/development/compilers/cudatoolkit/default.nix

```diff
@@ -304,5 +318,13 @@ in rec {
     gcc = gcc7;
   };
 
+  cudatoolkit_10_1 = common {
+    version = "10.1.243";
+    url = "https://developer.download.nvidia.com/compute/cuda/10.1/Prod/local_installers/cuda_10.1.243_418.87.00_linux.run";
+    sha256 = "0caxhlv2bdq863dfp6wj7nad66ml81vasq2ayf11psvq2b12vhp7";
+
+    gcc = gcc7;
+  };
+
   cudatoolkit_10 = cudatoolkit_10_0;
```

Comment:
We probably want to default to cudatoolkit_10_1 here.

Reply:
done